app/controllers/search_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SearchController < ApplicationController
+  # def index
```

Comment:
why is this line neccesary?

Reply:
will remove in the conflicting merge request